tslint.json

```diff
@@ -0,0 +1,46 @@
+{
+    "extends": ["tslint-react", "tslint-config-prettier"],
```

Comment:
You haven't installed `prettier`, so you shouldn't use this plugin. IIRC this plugin just auto-disables all rules that prettier will handle for you (so you won't get annoyed by the linter or get conflicts between linter and prettier), so you might be missing quite a lot of style lint warnings if you have this here.